portfolio/src/main/webapp/script.js

```diff
@@ -17,85 +17,87 @@ let MAX_COMMENTS = 5;
 
 /** Generate a random greeting */
 function generateRandomGreeting() {
-    //TODO use google translated to get greetings in different languages
-    const greetings =
-        ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //TODO use google translated to get greetings in different languages
+  const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-    // Pick a random greeting.
-    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-    return greeting;
+  // Pick a random greeting.
+  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  return greeting;
 }
 
 /* Update the MAX_COMMENTS */
 function updateNumComments() {
-    const userNumComments = document.querySelector("#numComments").value;
-    getUserComments(userNumComments);
+  const userNumComments = document.querySelector("#numComments").value;
+  getUserComments(userNumComments);
 }
 
 /* Retrieve user comments and display them */
 async function getUserComments(numComments=MAX_COMMENTS) {
-    try {
-        const response = await fetch(`/data?numComments=${numComments}`);
-        const data = await response.json();
+  try {
+    const response = await fetch(`/${await fetchBlobUrl()}?numComments=${numComments}`);
+    const data = await response.json();
 
-        const commentEl = document.querySelector("#user-comments");
-        if(typeof(commentEl) != 'undefined' && commentEl != null) {
-            commentEl.innerText = "";
-            // add a legend
-            commentEl.appendChild(createLegendEl());
+    const commentEl = document.querySelector("#user-comments");
+    if(typeof(commentEl) != 'undefined' && commentEl != null) {
+        commentEl.innerText = "";
 
-            for(let comment in data) {
-                commentEl.appendChild(createElement(data[comment]));
-            }
-            commentEl.style.display = "block";
+        for(let comment in data) {
+          commentEl.appendChild(createElement(data[comment]));
         }
-    } catch(err) {
-        console.log("There was an error loading comments!");
     }
+  } catch(err) {
+    console.log("There was an error loading comments!");
```

Comment:
You can use `console.error(...)` to give it error styling, and also because we tend to automatically flag any `console.log`s

Also, you can show the actual error by using `console.error('There was an error loading comments:', err);`

Reply:
> You can use `console.error(...)` to give it error styling, and also because we tend to automatically flag any `console.log`s
> 
> Also, you can show the actual error by using `console.error('There was an error loading comments:', err);`

Thank you for highlighting this out, I will correct it asap